erpnext/patches/v14_0/update_posting_datetime.py

```diff
@@ -0,0 +1,14 @@
+import frappe
+
+
+def execute():
+	frappe.reload_doc("Stock", "doctype", "Stock Ledger Entry")
```

Comment:
Not required in post model sync patches.

Reply:
Fixed